corehq/apps/users/forms.py

```diff
@@ -1177,7 +1177,7 @@ def __init__(self, *args, **kwargs):
         self.domain = kwargs.pop('domain')
         self.couch_user = kwargs.pop('couch_user')
         self.user_type = kwargs.pop('user_type')
-        if not(self.user_type not in [MOBILE_USER_TYPE, WEB_USER_TYPE]):
+        if self.user_type not in [MOBILE_USER_TYPE, WEB_USER_TYPE]:
```

Comment:
Thinking that user_types are passed internally within the code itself, so why is this check really necessary? When could the values be other than these?

Reply:
It's just defensive.

Reply:
Cool.

I think it just makes someone reading the code think that there can be other user_type which we are not supporting here or something like that. Fine to keep it as an extra safety net.

Reply:
nit: Though, it might be good to have it at the entry point(s) only? iirc, it's at many places, thinking if its not needed everywhere.

Reply:
I have mixed feelings about this - I like having less code, but also, why not be defensive everywhere? I'm going to leave this as-is for now.

Reply:
ya, i was just thinking over-defensive, if it was.